packages/extension-code-block/src/code-block.ts

```diff
@@ -113,6 +113,28 @@ export const CodeBlock = Node.create<CodeBlockOptions>({
           return this.editor.commands.clearNodes()
         }
 
+        return false
+      },
+      ArrowDown: () => {
+        const state = this.editor.state
+        const { from, to } = state.selection
+
+        if (from > 1 && from === to) {
+          let inCodeBlock = false
+          state.doc.nodesBetween(from - 1, to - 1, node => {
+            if (node.type.name === 'codeBlock') inCodeBlock = true
+          })
+
+          let nothingOnRight = true
+          state.doc.nodesBetween(from + 1, to + 1, node => {
+            if (node) nothingOnRight = false
+          })
+
+          if (inCodeBlock && nothingOnRight) {
+            return (this.editor.commands as any).setHardBreak()
```

Comment:
It's not good to rely on commands of another extensions. Not everyone is using the hard break extension so this can lead to errors. There is a core command called `exitCode()`. Can you try this instead?

Reply:
`exitCode` works nicely 👍